recipes/jinja2cpp/all/conanfile.py

```diff
@@ -89,7 +95,32 @@ def package(self):
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "jinja2cpp"))
         tools.rmdir(os.path.join(self.package_folder, "share"))
+        self._create_cmake_module_alias_targets(
```

Comment:
is there a PR for that opened already?

Reply:
I don't think so, you mean in conan?

Reply:
yes.

Reply:
there's a PR for this, I found this https://github.com/conan-io/conan/pull/6438

But there's another one Space and Madebr have commented on that I did not find

Reply:
@prince-chrismc not this PR (which has been reverted by the way).

There is an issue opened: https://github.com/conan-io/conan/issues/8360, but no PR AFAIK

Reply:
okay, I've created one: https://github.com/conan-io/conan/pull/8533